crates/control/src/game_controller_filter.rs

```diff
@@ -71,9 +89,47 @@ impl GameControllerFilter {
                 hulks_team_is_home_after_coin_toss: game_controller_state_message
                     .hulks_team_is_home_after_coin_toss,
             });
+
+            self.last_contact.insert(*source_address, time);
+            let on_cooldown = self
```

Comment:
Naming: Maybe "has cooled down" or also name the WHAT like "last audio message has cooled down" to make it even more clear.

Reply:
I renamed this variable.